app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
 
     buildTypes {
         debug {
-            applicationIdSuffix ".debug"
+            applicationIdSuffix ".debugcamerax"
```

Comment:
To facilitate having both the older app and the one with Camera X on the same device, I changed the debug package name. It would be reverted.